### PR TITLE
doc/tutorials/basics: fix example of aliases

### LINKED ISSUE #948: documentation: Modify the confusing tour/references/aliases page
**Documentation of aliases & let**

The following page of the doc refering to `aliases` is using the `let` construct:
https://cuelang.org/docs/tutorials/tour/references/aliases/

But to be honest I am not sure if the page is meant to introduce the concept of `aliases` or the `let` construct. These two are treated in the spec.md as two different concepts (with different use cases).

**Describe the solution you'd like**

I suggest to align the page with the spec.md to avoid confusion.

To the best of my knowledge a more idiomatic way to write the code snippet of the aliases page would be:
```
A=a: {
    d: 3
}
b: {
    a: {
        // A provides access to the outer "a" which would
        // otherwise be hidden by the inner one.
        c: A.d
    }
}
```

### PR DESCRIPTION
**Pull request by yujunz**
Fix #948

### CODE REVIEW

**Comment by myitcv** [comment]
Thanks for raising this. Let's first conclude in https://github.com/cuelang/cue/issues/948 exactly what changes we want to make. Because, as the issue suggests, we likely want different pages for aliases and let.

File: doc/tutorial/basics/4_references/30_aliases.txt [hunk 1]
```diff
@@ -14,8 +14,7 @@ Aliases are not members of a struct. They can be referred to only within the
 struct, and they do not appear in the output.
 
 -- alias.cue --
-let A = a  // A is an alias for a
-a: {
+A = a: { // A is an alias for a
```
Comment (mpvl) [inline doc/tutorial/basics/4_references/30_aliases.txt:17]: The current text above really describes let expressions, though.

Comment (yujunz) [reply doc/tutorial/basics/4_references/30_aliases.txt:15]: Updated text too.

**Comment by myitcv** [comment]
Hi @yujunz - thanks for creating this PR. 

CUE recently moved to a new organisation on GitHub: https://github.com/cue-lang/cue/issues/1078. Unfortunately we were unable to transfer this repository to the new organisation, and so pull requests will need to be re-created at https://github.com/cue-lang/cuelang.org.

Apologies for the inconvenience - we look forward to seeing you in CUE's new home. 

Thank you again for contributing to the CUE project!

**Pull request closed without merging**